AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -161,6 +162,47 @@ extension CardViewController {
             return holderNameItem
         }()
 
+        internal lazy var coBadgedCardItem: FormCoBadgedCardItem = {
+            let item = FormCoBadgedCardItem(
+                title: localizedString(
+                    .creditCardDualBrandTitle,
+                    configuration.localizationParameters
+                ),
+                subtitle: localizedString(
+                    .creditCardDualBrandDescription,
+                    configuration.localizationParameters
+                ),
+                selectableFormItems: [],
+                style: .init(
+                    title: configuration.style.sectionHeader,
+                    subtitle: configuration.style.footnoteLabel
+                )
+            )
+            item.style.subtitle.textAlignment = .left
+            item.identifier = ViewIdentifierBuilder.build(
+                scopeInstance: self,
+                postfix: "coBadgedCardItem"
+            )
+            return item
+        }()
+
+        internal func selectableFormItems(from brands: [CardBrand]) -> [SelectableFormItem] {
+            brands.map { brand in
+                let selectableItem = SelectableFormItem(
+                    title: brand.type.name,
+                    imageUrl: numberContainerItem.numberItem.cardTypeLogos.filter { $0.type == brand.type }.first?.url,
+                    isSelected: brand.type.rawValue == brands.first?.type.rawValue ? true : false,
```

Comment:
Just highlighting it here - provider ideally should just create entity and not decide on what is selected. It is business logic leaking to assembly layer. It's not a blocker for merge but it requires test coverage if possible. 

The use-case I read here: "as a user, whenever I enter co-branded card number, I want the first brand on the list to be auto-selected"